db/pgstorage/pgstorage.go

```diff
@@ -448,6 +448,12 @@ func (p *PostgresStorage) Set(ctx context.Context, key []byte, value [][]byte, r
 	return err
 }
 
+// BulkSet is similar to Set, but it inserts multiple key-value pairs into the db.
+func (p *PostgresStorage) BulkSet(ctx context.Context, rows [][]interface{}, dbTx pgx.Tx) error {
+	_, err := p.getExecQuerier(dbTx).CopyFrom(ctx, pgx.Identifier{"mt", "rht"}, []string{"key", "value", "root_id"}, pgx.CopyFromRows(rows))
```

Comment:
Why are you using copyfrom?? Are you copying the table to insert some rows??
Instead of doing this, I would suggest doing just something like this: 
```
INSERT INTO table_name (column_list)
VALUES
(value_list_1),
(value_list_2)
```
you can do a loop and depending on the number of elements in the array include more inserts on the query. I think it is easier and lighter fo the db. Also, doing it in this way you could avoid create `[][]interface{}` for each row and you could use an array of structs

Reply:
`copy` is faster than `multi inserts`, refer to https://www.postgresql.org/docs/current/populate.html